rds-deployment/azuredeploy.json

```diff
@@ -662,7 +675,7 @@
             "typeHandlerVersion": "2.11",
             "autoUpgradeMinorVersion": true,
             "settings": {
-              "modulesUrl": "[concat(variables('assetLocation'),'/Configuration.zip')]",
+              "modulesUrl": "[concat(parameters('_artifactsLocation'),'DSC/Configuration.zip',parameters('_artifactsLocationSasToken'))]",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#creating-uris for how to create the uris

Reply:
fixed